app/services/flickr_service.rb

```diff
@@ -0,0 +1,10 @@
+require 'net/http'
+
+class FlickrService
+  FLICKR_API_URL = 'https://api.flickr.com/services/rest/?method=flickr.photos.search&api_key='.freeze
+
+  def search(query)
+    uri = URI.parse("#{FLICKR_API_URL}#{ENV['FLIKR_API_KEY']}&tags=#{query}&per_page=24&format=json&nojsoncallback=1")
```

Comment:
Can we use [`URI::HTTP`](https://ruby-doc.org/stdlib-2.5.1/libdoc/uri/rdoc/URI/HTTP.html) and [`Hash#to_query`](https://apidock.com/rails/ActiveSupport/CoreExtensions/Hash/to_query)?